CMakeLists.txt

```diff
@@ -605,3 +605,33 @@ if(SENTRY_BUILD_SHARED_LIBS)
 	    "$<$<PLATFORM_ID:Android>:-Wl,-z,max-page-size=16384>"
 	)
 endif()
+
+if(WIN32 AND SENTRY_BUILD_SHARED_LIBS)
```

Comment:
Is this correct? Is there any scenario where this is true but resource files are not supported? Is this the right place to actually do this?

Reply:
It should be okay since the `MinGW` build didn't complain. @past-due, do you see a problem with version resource files in `MinGW`?

Reply:
Regarding the place in the CMake script, I would probably move the whole block to line 371.

Reply:
> It should be okay since the `MinGW` build didn't complain. @past-due, do you see a problem with version resource files in `MinGW`?

I think GCC (and MinGW) and Clang have both supported RC-files for a while. GCC via windres and Clang via llvm-rc.